components/authentication/pact-broker.yaml

```
@@ -27,3 +27,17 @@ roleRef:
   apiGroup: rbac.authorization.k8s.io
   kind: ClusterRole
   name: pact-broker-maintainer
+---
+kind: RoleBinding
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: pact-broker-maintainers
```

it would conflict with the other ClusterRoleBinding
```suggestion
  name: pact-broker-maintainers-view
```

---

Thank you